src/js/stores/DCOSStore.js

```diff
@@ -25,7 +25,9 @@ class DCOSStore extends EventEmitter {
     });
 
     this.data = {
-      marathon: new ServiceTree(),
+      marathon: {
+        tree: new ServiceTree()
```

Comment:
`services`? Why do we want to name this after the data structure it uses rather than what it represents?